src/Utils/CMakeLists.txt

```
@@ -0,0 +1,26 @@
+cmake_minimum_required(VERSION 3.16)
+
+project(Utils
+	LANGUAGES CXX
+	VERSION 0.0.1)
+
+qt_standard_project_setup()
+
+set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOMOC ON)
+set(CMAKE_AUTORCC ON)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+find_package(QT NAMES Qt6 Qt5 REQUIRED COMPONENTS Core)
+find_package(Qt${QT_VERSION_MAJOR} REQUIRED COMPONENTS Core)
+
+add_library(utils STATIC
```

```suggestion
qt_add_library(utils SHARED
```

Gibt glaube ich keinen Grund die STATIC zu machen. Gerade wenn die beiden anderen Module dagegen linken.

---

4cf884b